src/main/java/org/manifold/compiler/back/digital/smt2/ParenList.java

```diff
@@ -0,0 +1,44 @@
+package org.manifold.compiler.back.digital.smt2;
+
+import java.io.IOException;
+import java.io.Writer;
+import java.util.ArrayList;
+import java.util.List;
+
+import com.google.common.collect.ImmutableList;
+
+public class ParenList extends SExpression {
```

Comment:
It seems to me that this is just a wrapper around a list? Do we need an object just for this?